test_game_of_dbs.py

```diff
@@ -0,0 +1,101 @@
+from datetime import datetime
+
+from dateutil import parser
+
+import implement_me
+import pytest
+import datasets
+from elastic_handler import start_elastic, close_elastic
+from interface import Entry
+import ipaddr
+from time import sleep
+
+
+@pytest.fixture()
+def setup_data():
```

Comment:
Nice, but what about deleting all the data?

Reply:
Done